### PR TITLE
More details on build dependencies and vaapi on README. Workaround for blackscreen under Wayland

### PR DESCRIPTION
**Pull request by lovenemesis**
Add a bit more details regarding to dependencies and vaapi, using Fedora as example.

Also a workaround for #71 for users running Chiaki under Wayland.

Many thanks to this great work!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,8 +40,21 @@ You can download them [here](https://github.com/thestr4ng3r/chiaki/releases).
...
+
+```
+sudo dnf install cmake make qt5-qtmultimedia-devel qt5-qtsvg-devel qt5-qtbase-gui ffmpeg-devel opus-devel openssl-devel python3-protobuf protobuf-c protobuf-devel qt5-rpm-macros SDL2-devel
+```
+
+In order to utilize hardware decoding, necessary VA-API component needs to be installed separately depending on your GPU. For example on Fedora:
+
+* **Intel**: `libva-intel-driver`(majority laptop and desktop) OR `libva-intel-hybrid-driver`(most netbook with Atom processor)
+* **AMD**: Already part of default installation
+* **Nvidia**: `libva-vdpau-driver`
```
Comment (thestr4ng3r) [inline README.md:56]: Please put all this Fedora-specific info into the new `doc/platform-build.md` file, above the Windows section (Windows has the lowest priority).

File: gui/chiaki.desktop [hunk 2]
```diff
@@ -2,6 +2,6 @@
 Type=Application
 Name=Chiaki
 Comment=PlayStation 4 Remote Play Client
-Exec=chiaki
+Exec=env QT_QPA_PLATFORM=xcb chiaki
```
Comment (thestr4ng3r) [inline gui/chiaki.desktop:5]: This is not needed anymore as the EGL (Wayland) support is now fixed.

File: README.md (hunk 1 above)
Comment (lovenemesis) [reply README.md]: Done. Moved above instruction to platform doc.

File: gui/chiaki.desktop (hunk 2 above)
Comment (lovenemesis) [reply gui/chiaki.desktop]: Great news! I revert this workaround then.

**Pull request merged**